recipes/cyipopt/meta.yaml

```diff
@@ -0,0 +1,73 @@
+{% set name = "cyipopt" %}
+{% set version = "0.1.7" %}
+{% set sha256 = "e62d4265743078c60f0af1426218310a9fbc1b6d0d0e60879e172d569dd3343d" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: v{{ version }}.tar.gz
+  url: https://github.com/matthias-k/cyipopt/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  # TODO: Once ipopt is available for Windows in conda forge, remove the
+  # following line.
+  skip: True  # [win]
+  # NOTE : Set CC to force gcc on OSX.
+  script: CC=gcc python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - gcc
+    - pkg-config  # [osx]
+    - python
+    - ipopt
+    - setuptools
+    - cython
+    - numpy 1.11*
+    - scipy
+    - six
+    - future
+  run:
+    - python
+    - ipopt
+    - numpy 1.11*
+    - scipy
+    - six
+    - future
+
+test:
+  requires:
```

Comment:
Please remove all this section. It is redundant because `test` already installs the `run` dependencies.

```yaml
  requires:
    - python
    - ipopt
    - numpy 1.11*
    - scipy
    - six
    - future
```